maintainers/maintainer-list.nix

```diff
@@ -2479,6 +2479,11 @@
     github = "jonringer";
     name = "Jonathan Ringer";
   };
+  jorise = {
```

Comment:
Please include this as a separate commit, before the commit adding the package.

Reply:
A seperate commit was added.